src/stale-prs.ts

```diff
@@ -340,6 +342,28 @@ export class StalePrFinder {
 
     return ret;
   }
+
+  /**
+   * Returns true if the PR has been reviewed or commented on by a core member.
+   */
+  private async memberReviewed(pull_number: number): Promise<Boolean> {
+    const reviews = (await this.client.paginate(this.client.rest.pulls.listReviews, { ...this.repo, pull_number }));
+
+    const comments = await this.client.paginate(this.client.rest.issues.listComments, {
+      ...this.repo,
+      issue_number: pull_number,
+    });
```

Comment:
Will all reviews leave comments? If so do we need to look through the reviews?

Reply:
Nit: these lines are formatted differently.

Reply:
Ya I wasn't 100% sure if `issues.listComments` would also include reviews hence the separate check, though I don't think it'd hurt to keep the check for reviews.